src/values/mod.rs

```diff
@@ -47,7 +47,7 @@ pub use crate::values::traits::AsValueRef;
 pub use crate::values::traits::{AggregateValue, AnyValue, BasicValue, FloatMathValue, IntMathValue, PointerMathValue};
 pub use crate::values::vec_value::VectorValue;
 
-#[llvm_versions(12.0..=latest)]
+#[llvm_versions(12.0..=16.0)]
 use llvm_sys::core::LLVMIsPoison;
```

Comment:
Is poison no longer supported in LLVM17?

Reply:
Looks like it still is, do I'll revert that later tonight: https://docs.rs/llvm-sys/latest/llvm_sys/?search=LLVMIsPoison

Reply:
Actually that import wasn't even being used so I got rid of it.